users/vieting/experiments/switchboard/ctc/feat/network_helpers/specaug_configurable.py

```diff
@@ -159,12 +160,18 @@ def get_masked():
             tf.math.floor(config["freq_mask_max_proportion"] * tf.cast(tf.shape(x)[data.feature_dim_axis], tf.float32)),
             tf.int32,
         )
-
+        max_time_num_seq_len = tf.cast(
+            tf.math.floordiv(
+                tf.cast(tf.shape(x)[data.time_dim_axis], tf.float32),
+                max_time_num_seq_len_divisor * tf.cast(time_mask_max_size, tf.float32),
```

Comment:
Shouldn't it be something like 1 / this line?

Reply:
Yes, I missed that.